src/three/PNTSLoader.js

```diff
@@ -43,7 +43,26 @@ export class PNTSLoader extends PNTSLoaderBase {
 				} );
 
 				const geometry = new BufferGeometry();
-				geometry.setAttribute( 'position', new BufferAttribute( POSITION, 3, false ) );
+				if ( POSITION ) {
+
+					geometry.setAttribute( 'position', new BufferAttribute( POSITION, 3, false ) );
+
+				} else if ( POSITION_QUANTIZED ) {
+
+					const decodedPositions = new Float32Array( POINTS_LENGTH );
+					for ( let i = 0; i < POINTS_LENGTH / 3; i ++ ) {
```

Comment:
Is this correct? The `POSITION_QUANTIZED` field should be 3x the length of `POINTS_LENGTH` since it holds vector3s. It seems when using `POINTS_LENGTH` we shouldn't be dividing by 3. I'm looking at the spec here:

https://github.com/CesiumGS/3d-tiles/tree/main/specification/TileFormats/PointCloud#global-semantics

Reply:
Oh my god, so sorry for this, I should've double checked it before making the commit, I've fixed it now

Reply:
Heh - that's what code reviews are for 😁